### ISSUE TITLE
[WIP][P3.1][Sourcing][Accessibility][Pretest][Table sort]Sorting the table is not possible with the keyboard, as the context menu does not receive the keyboard focus.

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10819]
#### Is this a bug, enhancement, or feature request?
bug

#### Describe your proposal.
Sorting the table is not possible with the keyboard, as the context menu does not receive the keyboard focus automatically on opening the popover.

expectation:After opening the context menu, the focus should automatically be set to the first menu item. It should be possible to navigate within the menu using the arrow keys.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
Angular: 15
core/platform: 0.43.14

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/table/basic#p13-table-sort

**Comment by nikolay-kolarov** [comment]
Hi @I543348 ,

I checked this https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/table/basic and it seems to work already.
Could you please retest and provide more info, if this is still reproducible on your side?

Best Regards,
Nikolay

**Comment by I543348** [comment]
Hi @nikolay-kolarov , I tested it on https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/table/basic#sortable , works fine for me .
@sagar7259 waiting for Sagar to confirm the same and we are good here.
Thanks.
